amenity-points.mss

```diff
@@ -1867,6 +1879,7 @@
   [feature = 'man_made_silo'][zoom >= 18],
   [feature = 'man_made_communications_tower'][zoom >= 17],
   [feature = 'man_made_telescope']["telescope:type" = 'radio'][zoom >= 16],
+  [feature = 'man_made_telescope'][zoom >= 17],
```

Comment:
If we use optical vs all the other for icons, I guess name rendering should also check it, not radio vs the rest.